app/i18n/messages/fi.json

```diff
@@ -499,6 +499,7 @@
   "Reservation.stateLabelConfirmedButNotPaid": "Hyväksytty, mutta ei maksettu",
   "Reservation.stateLabelDenied": "Evätty",
   "Reservation.stateLabelRequested": "Käsiteltävänä",
+  "Reservation.stateLabelInvoiceRequested": "Lasku käsiteltävänä",
```

Comment:
"Pyydetty maksua laskulla" would probably be better here, since the invoice is not being processed yet.

Reply:
Yes it's a bit confusing as we use "Requested" in English which for both cases.

Reply:
Yep